source/wp-content/themes/wporg-news-2021/sass/components/local-header/_breadcrumbs.scss

```
@@ -0,0 +1,33 @@
+.local-header {
+    // Breadcrumbs show the lineage of pages based on the current
+    // page. It contains a link for each parent.
+    .local-header__breadcrumb {
+        display: flex;
+        align-items: center;
+        min-height: var(--bar-min-height);
+        color: var(--bar-text-color);
+
+        // The current item is highlighted.
+        &-current {
```

```suggestion
        .local-header__breadcrumb-current {
```

#207 changed most of the partial selectors, but it looks like this one was missed.